src/game/output.c

```diff
@@ -320,7 +346,6 @@ static const int16_t *Output_CalcRoomVertices(const int16_t *obj_ptr)
             int32_t depth = zv_int >> W2V_SHIFT;
             if (depth > Output_GetDrawDistMax()) {
                 m_VBuf[i].g = 0x1FFF;
-                clip_flags |= 16;
```

Comment:
1. Can we restore the original code and fence it with a condition `if (!g_Config.enable_skybox)`?
2. Do we know how this is going to affect the rendering performance – do we now draw the entire level geometry, or do we still clip unseen rooms?

Reply:
1. Done - I've referenced the object loaded state though in case a custom level has the object, but the player disables the setting, as that might come across as a level issue. LMK if this looks OK. My idea was the config option only controls the injection.
2. Performance seems good to me. Yes, `Room_SetBounds` will cull rooms that are too far away anyway, so the above only applies to rooms in the "immediate" vicinity.
https://github.com/lahm86/TR1X/blob/develop/src/game/room_draw.c#L101